source/collaborate/convert-public-channels.rst

```diff
@@ -53,13 +53,18 @@ You must be a system admin or team admin to convert public channels to private c
 Convert private channels to public channels
 -------------------------------------------
 
-Due to potential security concerns with sharing private channel history, only system admins can convert private channels to public channels.
+Due to potential security concerns with sharing private channel history, only system admins can convert private channels to public channels. 
+
+.. note::
+
+  Team admins may be granted permissions to convert private channels to public channels via the `API <https://api.mattermost.com/#tag/channels/operation/UpdateChannelPrivacy>`_ or :ref:`mmctl channel modify command <manage/mmctl-command-line-tool:mmctl channel modify>` when configured in the :ref:`permission scheme <onboard/advanced-permissions:permissions structure>`.
```

Comment:
@cwarnermm checking the implementation, this is not entirely the case. 

Convertion from private to public can happen from team admins, sysadmins, and users with delegated granular administration roles with write access to User Management Channels and Groups. (`PermissionSysconsoleWriteUserManagementChannels` and `PermissionSysconsoleWriteUserManagementGroups`). 

Teams admins are granted this permission by default but a system admin can configure to restrict this from team admins

Reply:
cc. @JulienTant to verify

Reply:
Yea you are correct @esarafianou